Foundation/NSURLResponse.swift

```diff
@@ -233,7 +233,8 @@ open class HTTPURLResponse : URLResponse {
     /// This property is intended to produce readable output.
     override open var description: String {
         var result = "<\(type(of: self)) \(Unmanaged.passUnretained(self).toOpaque())> { URL: \(url!.absoluteString) }{ status: \(statusCode), headers {\n"
-        for(key, value) in allHeaderFields {
+        for(aKey, aValue) in allHeaderFields {
+            let (key,value) = (aKey as! String, aValue as! String) //allHeaderFields is either assigned a [String:String] or [:]
```

Comment:
If we allow the header fields to be anything besides a string as part of the API, then shouldn't we use `as?` here? Otherwise we'll just assert in some cases.